docs/en/week08/practicum08.sbv

```diff
@@ -137,7 +137,7 @@ two macro modules I guess and so the
 input get goes inside an encoder
 
 0:02:09.989,0:02:13.560
-which gives us a code so H which was
+which gives us a code so h which was
```

Comment:
0:01:13.920,0:01:17.900
this network out to layer neural net>>...a two layer...